autoload/fern/internal/viewer/auto_duplication.vim

```diff
@@ -1,5 +1,5 @@
 function! fern#internal#viewer#auto_duplication#init() abort
-  if g:fern#disable_viewer_auto_duplication || !g:fern#disable_drawer_universal
+  if g:fern#disable_viewer_auto_duplication || g:fern#disable_drawer_tabpage_isolation
```

Comment:
You need to check if the viewer is drawer or not. See the original PR

Reply:
Done